dna/modify.go

```diff
@@ -96,6 +96,11 @@ func Complement(bases []Base) {
 	}
 }
 
+func ComplementSingleBase(base Base) Base {
```

Comment:
This looks like a public version of the "complement" function.  Can we just rename the existing "complement" function to "ComplementSingleBase"?